tanner/api/api.py

```diff
@@ -2,22 +2,44 @@
 import logging
 import operator
 import aioredis
+import psycopg2
+from collections import ChainMap
+from tanner.utils.attack_type import AttackType
 
 
 class Api:
-    def __init__(self, redis_client):
+    def __init__(self, redis_client, pg_client):
```

Comment:
Do we still need redis?

Reply:
We need it till the time we separate the analyses part and the deletion part.

Reply:
We need it till the time we separate the analyses part and the deletion part.